ckanext/data_qld_theme/templates/package/comment_list.html

```diff
@@ -9,6 +9,10 @@
 {% set depth = 0 %}
 {% set notification_actions_enabled = userobj and h.comment_notification_recipients_enabled() %}
 
+{%- if h.user_can_manage_comments(content_type, pkg_id) -%}
```

Comment:
This should gracefully handle the situation where the helper function does not exist.

Reply:
Is this really necessary?

The same template helper function has been used in the `ckanext-data-qld-theme/ckanext/data_qld_theme/templates/package/comment_list.html` template override since it's inception

The `user_can_manage_comments` helper function has existed in the `qld-gov-au` fork of the YTP comments extension since 14 Jun 2019

I wouldn't expect that you'd revert your version of YTP comments to a version before all the enhancements were made for data.qld back in 2019

Reply:
@salsa-nathan The more robust the implementation is, the more easily we can adapt it to the Publications site, which doesn't use the Comments extension.

Reply:
@ThrawnCA `user_can_manage_comments` is only used in the `comment_list.html` template override in this extension. If you're not using the comments extension on the Publications site, you don't need this template override in your theme.